recipes/symopt/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "symopt" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/spcornelius/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 24335e1ed3306a5b30f9f7eab97b8bb66b562c625a260c32d0f8caaf2a485ec9
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: python -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - python >= 3.6
+    - setuptools
+    - pip
+  run:
+    - python >= 3.6
+    - numpy
+    - sympy
+    - scipy
+    - cyipopt
+    - orderedset
+
+test:
+  imports:
+    - {{ name }}
+  requires:
```

Comment:
that is odd
do you need ``pytest`` and ``fortran`` ? You can drop it, however, if you need it to run this project you should add it as ``run`` dependency

Reply:
Hi Marcelo. The reason for the above arrangement is that one of the available SymPy backends for code generation requires fortran. However, there are other such backends (including the default setting in this package) that do not require fortran. As such, a user could perfectly well install and run this without fortran, but fortran is strictly required to run (all of) tests.

Does that make sense?